recipes/ftfy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ftfy" %}
+{% set version = "4.3.1" %}
+{% set sha256sum = "60af7f4566a966fcb9681f3c93814017e981f5bbfffc2f5f33aadf8238101f4b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    # Setuptools is required for pkg_resources.
+    - setuptools
```

Comment:
I added it because the code uses `pkg_resources.resource_string` to load package data. But after creating a fresh env I noticed `setuptools` is always installed, so I'm leaving it out.